cluster_experiments/random_splitter.py

```diff
@@ -538,3 +537,32 @@ def from_config(cls, config):
             treatments=config.treatments,
             treatment_col=config.treatment_col,
         )
+
+
+class FixedTreatmentClustersSplitter(ClusteredSplitter):
+    def __init__(self, cluster_cols: List[str], n_treatment_clusters: int):
```

Comment:
can you add some docstring explaining what this does and why is it different than other splitters?